docs/content/latest/benchmark/performance-troubleshooting.md

```diff
@@ -108,10 +105,26 @@ W0325 06:47:13.033341 116514816 inbound_call.cc:208] Trace:
 0325 06:47:13.032168 (+    62us) inbound_call.cc:125] Queueing success response
 ```
 
+## Viewing real-time logs
+
+You can view logs of Yugabyte processes at a particular node (e.g. 127.0.0.1) at various ports:
+
+| Description | Address
+-------------|-----------|
+Master Metrics | 127.0.0.1:7000
+TServer Metrics | 127.0.0.1:9000
+Yedis Metrics | 127.0.0.1:11000
+YCQL Metrics | 127.0.0.1:12000
+YSQL Metrics | 127.0.0.1:13000
+Per-Tablet, JSON Metrics | 127.0.0.1/metrics
+Per-Table, Prometheus Metrics | 127.0.0.1/prometheus-metrics
+
 ## Turning RPC tracing on
 
 To turn tracing on for all RPCs, not just the slow ones, use the `rpc_dump_all_traces` gflag.
 
+    $ ./yb-ts-cli --server_address=localhost:9100 set_flag rpc_dump_all_traces 1
```

Comment:
cc @amitanandaiyer don't we also need some `--enable_tracing` ?

Reply:
yes. We need enable_tracing to be turned on for "collecting traces".

Once the collection is turned on. We can print them out based on latency/sampling or all. I've personally never used all. I tend to use the latency based, or the sampling based printing.

Reply:
we don't want to print so many traces ... that printing the traces itself slows down the system/logging.

depending on the scenario, one must adjust the latency-threshold/print_every_n frequency to make sure we are not disturbing the state of the system too much.